server/package.json

```diff
@@ -8,6 +8,9 @@
     "dotenv": "^16.0.1",
     "express": "^4.18.1",
     "http": "^0.0.1-security",
+    "mongodb": "^4.8.0",
+    "mongoose": "^6.4.5",
```

Comment:
If we are not using `mongoose`, then better we should remove it from node dependencies.